examples/torch/common/utils.py

```diff
@@ -189,8 +189,10 @@ def create_code_snapshot(root, dst_path, extensions=(".py", ".json", ".cpp", ".c
 
 
 def print_args(config, logger=default_logger):
+    logger.info("\nConfiguration parameters:")
```

Comment:
Somewhere we use double quotes, somewhere single quotes. It doesn't affect functionality (usually) and generally looks applicable. But the eye clings to it, in the PR, where we change the logging messages. I'm not advocating changing this, just FYI.

Reply:
Actually we have a specific style defined by the guideline, but I find it hard to enforce if we don't have a clean state to start from, and going through the code just to fix all the quotes, even automatically, has always been deprioritized.